install.sh

```diff
@@ -8,6 +8,7 @@ is_executable () {
 
 alias errcho='>&2 echo'
 npm_needs_sudo=''
+needs_python_env='false'
```

Comment:
`-z` below returns [true if the length is zero](http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html) but `$needs_python_env` is never empty.

Instead, you can use something like this:
```
hash python2 2>/dev/null && PYTHON='python2'
```

If you want, you can refactor the lines below with `sudo` to be less repetitive: (untested)
```
install_slap() { npm install -g slap; }
[ -z $npm_needs_sudo ] && install_slap=(sudo "${install_slap[@]}")
hash python2 2>/dev/null && PYTHON='python2'
install_slap
```

Feel free to change the whole file even! I slapped it up in a rush, and I'm not the best with shell scripting.